static/app/views/replays/deadRageClick/deadClickList.tsx

```diff
@@ -0,0 +1,119 @@
+import {
+  DeadRageSelectorItem,
+  DeadRageSelectorQueryParams,
+} from 'sentry/views/replays/types';
+
+function getAriaLabel(str: string) {
+  const pre = str.split('aria=')[1];
+  if (!pre) {
+    return '';
+  }
+  return pre.substring(0, pre.lastIndexOf(']'));
+}
+
+function mappedData(data): DeadRageSelectorItem[] {
+  return data.map(d => {
+    return {
+      count_dead_clicks: d.count_dead_clicks,
+      dom_element: d.dom_element,
+      element: d.dom_element.split(/[#.]+/)[0],
+      aria_label: getAriaLabel(d.dom_element),
+    };
+  });
+}
+
+interface Props extends RouteComponentProps<{}, {}, DeadRageSelectorQueryParams> {}
+
+export default function DeadClickList({location}: Props) {
+  const organization = useOrganization();
+  const hasDeadCicks = organization.features.includes(
+    'session-replay-rage-dead-selectors'
+  );
+
+  const {isLoading, isError, data, pageLinks} = useDeadRageSelectors({
+    per_page: 50,
+    sort: '-count_dead_clicks',
+  });
+
+  return hasDeadCicks ? (
+    <SentryDocumentTitle
+      title={t('Top Selectors with Dead Clicks')}
+      orgSlug={organization.slug}
+    >
+      <Layout.Header>
+        <Layout.HeaderContent>
+          <Layout.Title>
+            {t('Top Selectors with Dead Clicks')}
+            <PageHeadingQuestionTooltip
+              title={t('See the top selectors your users have dead clicked on.')}
+              docsUrl="https://docs.sentry.io/product/session-replay/replay-page-and-filters/"
+            />
+          </Layout.Title>
+        </Layout.HeaderContent>
+      </Layout.Header>
+      <PageFiltersContainer>
+        <Layout.Body>
+          <Layout.Main fullWidth>
+            <LayoutGap>
+              <PageFilterBar condensed>
+                <ProjectPageFilter resetParamsOnChange={['cursor']} />
+                <EnvironmentPageFilter resetParamsOnChange={['cursor']} />
+                <DatePageFilter alignDropdown="left" resetParamsOnChange={['cursor']} />
+              </PageFilterBar>
+              <SelectorTable
+                data={mappedData(data)}
+                isError={isError}
+                isLoading={isLoading}
+                location={location}
+                clickType="count_dead_clicks"
+                clickCountColumn={{key: 'count_dead_clicks', name: 'dead clicks'}}
+              />
+            </LayoutGap>
+            {
+              <PaginationNoMargin
+                pageLinks={pageLinks}
+                onCursor={(cursor, path, searchQuery) => {
+                  browserHistory.push({
+                    pathname: path,
+                    query: {...searchQuery, cursor},
+                  });
+                }}
+              />
+            }
+          </Layout.Main>
+        </Layout.Body>
+      </PageFiltersContainer>
+    </SentryDocumentTitle>
+  ) : (
+    <Layout.Page withPadding>
+      <Alert type="warning">{t("You don't have access to this feature")}</Alert>
+    </Layout.Page>
+  );
```

Comment:
[logaf](https://blog.danlew.net/2020/04/15/the-logaf-scale/) level -> [m]edium: This ternary makes this condition a bit hard to read since the "primary" view (when hasDeadClicks = true) is so long, and this content gets pushed down. you'd have to read down, see else condition of the ternary, and then trace it back up.

I would probably make an explicit if condition for `!hasDeadClicks, and then the main content can be returned without any conditions (and un-nested!).

Reply:
no one noticed that i spelled it `hasDeadCicks` oops